librz/util/bitvector.c

```diff
@@ -4,6 +4,7 @@
 #include "rz_util.h"
 #include <stdlib.h>
 #include <stdio.h>
+#include <limits.h>
```

Comment:
```suggestion
#include <rz_types.h>
```

`limits.h` should be included already there.